x-pack/docs/en/security/securing-communications/node-certificates.asciidoc

```diff
@@ -88,12 +88,33 @@ Use the <<certutil-http,`elasticsearch-certutil http`>> command:
 
 [source,shell]
 ----
-bin/elasticsearch-certutil http
+./bin/elasticsearch-certutil http
 ----
 
 This command guides you through the process of generating the appropriate
 certificates for use in {es} and {kib}. If you created a CA for your cluster,
 you can re-use it by supplying its location when prompted.
+
+NOTE: The first question that the `elasticsearch-certutil` tool prompts you
+with is whether you want to generate a Certificate Signing Request (CSR).
+Answer `y` if you want to sign your own certificates, or `n` if you want to
+sign certificates with a central CA.
```

Comment:
I'm not sure that this terminology would clarify which choice to make - that is, "sign your own certificates" isn't particularly clear, and could mean "I want to do it as part of this process" (which would actually mean you should pick `n`) or "I handle it myself by sending it to someone in my organisation" (which is `y`).

I would go with something more like this:
```suggestion
Answer `y` if you need to have your certificates signed by someone else
(such as an internal security team, or a commercial certificate authority),
or `n` if you want to sign certificates with a CA that you have direct access to.
```
It feels a bit wordy though, and "direct access" might not be clear enough.

Reply:
I like the distinction that you incorporated. I think we can break these ideas into separate sentences:
>Answer `y` if a trusted authority, such as in internal security team or a commercial certificate authority, will sign your certificates. Answer `n` if you have access to a trusted CA and will sign your own certificates.